README.md

```diff
@@ -1,8 +1,118 @@
+
+#### Distributing configuration  
+##### Distributing to SASS  
+Each property of `config.data` specified in `config.layerAllocations.sass` is eventually written as a SASS file to  `/source/css/sass/derived-from-config/_variables--[propertyname].sass`. Each of these files contains the SASS variables describing the config for that property. Looking at the `breakpoint` example again, this config  
+  
+```  
+// specified in a config file  
+config.data.breakpoint = {site: {}};  
+config.data.breakpoint.site.small = 480;  
+config.data.breakpoint.site.medium = 730;  
+  
+config.layerAllocations.sass = ['breakpoint'];  
+```  
+  
+generates this file:  
+```  
+// /source/css/sass/derived-from-config/_variables--breakpoint.sass  
+$breakpoint-site-small: 480;  
+$breakpoint-site-medium: 730;  
+```   
+##### Distributing to JavaScript  
+Each property of `config.data` specified in `config.layerAllocations.jss` is eventually written to `/source/js/derived-from-config/configForJs.json`.  Looking at the `breakpoint` example again, this config:    
+  
+```js  
+// specified in a config file  
+config.data.breakpoint = {site: {}};  
+config.data.breakpoint.site.small = 480;
+config.data.breakpoint.site.medium = 730;  
+config.layerAllocations.js = ['breakpoint'];
+```  
+  
+adds this into `configForJs.json`:  
+```  
+// /source/js/derived-from-config/configForJs.json  
+...  
+{"breakpoint":{"site":{"small":480,"medium":730}}}  
+...  
+```  
+  ##### Distributing to templates
+  [Not yet implemented]
+  
+#### Modifying configuration
+##### Default configuration  
+Default configuration is supplied by `/libero-config/config--libero-default.js`. If this is sufficient, the rest of this section may be safely ignored.  
+  
+##### Changing configuration  
+Do not change the contents of the default config file `/libero-config/config--libero-default.js` directly.  
+  
+Any changes to the configuration should be effected by placing one or more custom configuration files into `/libero-config/`, and registering the file name(s) in `/libero-config/configRegister.json`. Any files listed here are loaded as config files. The order of the files listed  defines their load order. This is important when namespace clashes occur: when this happens the clashing name that was loaded last wins. This is how specific configuration properties are overridden.  
+  
+##### Swapping out configuration wholesale  
+Supply your own config file(s), add appropriate references to `/libero-config/configRegister.js`, and remove mention of `configs--libero-default.js` from `/libero-config/configRegister.js`.   
+  
+##### Keep default configuration but augment or override some of its properties  
+Supply your own config file(s), add appropriate references to `/libero-config/configRegister.js`.
```

Comment:
Probably don't actually want to support this? We know that we want to support extensions of the pattern library, but forking isn't really the way.

Reply:
Something to think about. Not sure just at the moment how else we might handle extensions. Will ponder.

Reply:
Will remove documentation, but leave implementation in for now, undocumented & so subject to change.